sdcm/nemesis.py

```diff
@@ -2123,6 +2123,17 @@ def disrupt_network_block(self):
             time.sleep(wait_time)
         finally:
             self.target_node.traffic_control(None)
+            self._wait_all_nodes_un()
```

Comment:
Can we reuse self.cluster.check_nodes_up_and_normal()?

Reply:
yes, we can and PR was already updated.